cmds/task/fetchers.go

```diff
@@ -58,7 +58,45 @@ func runStatus(credentials *tcclient.Credentials, args []string, out io.Writer,
 	fmt.Fprintln(out, getRunStatusString(s.Status.Runs[runID].State, s.Status.Runs[runID].ReasonResolved))
 	return nil
 }
+// confirmMsg displays confirmation message when --confirm is used
+func confirmMsg(command string, credentials *tcclient.Credentials, args []string)string {
+
+	q := makeQueue(credentials)
+	taskID := args[0]
+
+	c, _ := q.Status(taskID)
+	run := c.Status.Runs[len(c.Status.Runs)-1]
+
+	t, _ := q.Task(taskID)
+
+	fmt.Println(command, t.Metadata.Name ,"taskid:", taskID ,"(state:",run.State,"). Are you sure you want to proceed with given command? (y/N)")
```

Comment:
This would be better with `fmt.Printf` and format specifiers.  If you leave the \n off the end, then the prompt will be on the last line, which is normal for confirmation prompts.  Something like `fmt.Printf("%s %s taskId: %s (state: %s). Are you sure you want to proceed? ", command, t.Metadata.Name, taskId, run.State)`.  Note the space after the `?`.